ui/packages/consul-ui/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "consul-ui",
+  "name": "@hashicorp/consul-ui",
   "version": "2.2.0",
```

Comment:
Related to https://github.com/hashicorp/consul/pull/21378#issuecomment-2195832829

Reply:
Upgrading to 2.2.1 wouldn't hurt, but in practice this is a local-only package that hasn't had its version upgraded [in six years](https://github.com/hashicorp/consul/pull/4326) so I'm reluctant to buck that trend/